packages/template-graphql/README.md

```diff
@@ -4,7 +4,7 @@
 
 **Please see the [main Hops Readme](../../DOCUMENTATION.md) for general information and a Getting Started Guide.**
 
-This is a small example application showing [hops](https://github.com/xing/hops) in action. It demonstrates how to use hops with React, GraphQL and Jest.
+This is a small example application showing [hops](../..) in action. It demonstrates how to use hops with React, GraphQL and Jest.
```

Comment:
```suggestion
This is a small example application showing [hops](../../DOCUMENTATION.md) in action. It demonstrates how to use hops with React, GraphQL and Jest.
```

Just `../..` does not work as it points to `https://github.com/xing/hops/blob/master`